### DISCUSSION TITLE
Adding mergify

### DISCUSSION THREAD

**Discussion by hasnainmakada-99** [reactplay/react-play#556] [Ideas]
Hey everyone I was thinking about that what if we can automate the whole process of merging prs and closing of issues with the help of mergify, Will it work with reactplay ?

**Comment by atapas** [comment]
> Hey everyone I was thinking about that what if we can automate the whole process of merging prs and closing of issues with the help of mergify, Will it work ?

Great Idea!

A couple of questions

- Is it free? If PRO plan is there, will it be available for an OSS like us for free? Like how Nhost and Netlify helping us with?
- Is there a pilot done before we deploy.

I had explored mergify a while back, would like to know its potential.

btw @hasnainmakada-99 out Discord got the forum feature now.. we can discuss there too. :)

**Reply by hasnainmakada-99** [reply]
Lets discuss on discord
